docs/install.md

```diff
@@ -1,50 +1,71 @@
-# Getting Started
+# Quick Start Guide
 
- In this tutorial you'll create a full Kubernetes cluster with just one node including both the controller and the worker. This is well suited for environments where the high-availability and multiple nodes are not needed. This is the easiest install method to start experimenting k0s.
+ To help ease you into k0s installation practices Mirantis offers this Quick
```

Comment:
Remove "Mirantis" from all places as this is an open source project.

Reply:
Understood.